src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Application.cs

```diff
@@ -969,6 +969,31 @@ ResourceDictionary IHaveResources.Resources
             set { Resources = value; }
         }
 
+        /// <summary>
+        /// The ThemeMode property is used to set the Fluent theme mode of the application.
+        /// Setting this property controls if Fluent is loaded in Light, Dark or System mode. 
+        /// It also controls the application of backdrop and darkmode on window.
+        /// The four values for the ThemeMode enum are :
+        ///     <see cref="ThemeMode.None"/> - No Fluent theme is loaded.
+        ///     <see cref="ThemeMode.System"/> - Fluent theme is loaded based on the system theme.
+        ///     <see cref="ThemeMode.Light"/> - Fluent theme is loaded in Light mode.
+        ///     <see cref="ThemeMode.Dark"/> - Fluent theme is loaded in Dark mode.
+        ///
+        /// These values are predefined in <see cref="ThemeMode"/> struct
+        /// The default value is <see cref="ThemeMode.None"/>.
+        /// </summary>
+        /// <remarks>
+        ///     <see cref="ThemeMode"/> and <see cref="Resources"/> are designed to be in sync with each other.
+        ///     Syncing is done in order to avoid UI inconsistencies, where the window is in dark mode but the 
+        ///     controls within are in light mode or vice versa.
```

Comment:
Although this is self explanatory, but writing just about dark and light might be confusing as syncing also ensures consistency with `none` and `system` mode.

Possible suggestion: Syncing is done in  order to avoid UI inconsistencies, for example, if a window is in dark mode but controls within are in light mode.